docs/source/getting-started/content/proc_accessing-the-openshift-cluster.adoc

```diff
@@ -86,4 +86,6 @@ The `machine-config` and `marketplace` cluster operators are expected to report
 The `monitoring` cluster operator is expected to report `Unknown` availability.
 ====
 +
+  . The link:https://docs.openshift.com/container-platform/4.1/applications/projects/working-with-projects.html[OpenShift documentation] covers the creation of projects and applications.
+
```

Comment:
Actually not the right place for this, as this would be the last step in the "using oc" section, but it's also needed in the "using web console" case. Dunno if I should duplicate it, or put it somewhere else?

Reply:
Also note the "4.1" in the URL, but 4.2 documentation does not seem public yet.

Reply:
@cfergeau better to use https://docs.openshift.com/container-platform/latest/applications/projects/working-with-projects.html it should auto redirect to 4.2 one once it is released, right now it points to 4.1